lib/cli/main.ex

```diff
@@ -0,0 +1,7 @@
+defmodule TicTacToe.CLI.Main do
+  alias Mix.Shell.IO, as: Shell
+
+  def start_game do
+    Shell.info("Welcome to Tic Tac Toe!")
```

Comment:
Curious about your decision to use `Mix.Shell.IO` instead of Elixir's `IO`? 🤔

Reply:
Gotcha 👍🏼 FWIW I haven't seen `Mix.Shell.IO` used on any client projects, but I'll leave that up to you

Reply:
I'll change it back so I get used to following client processes! Thanks